docs/user-guide/conditional-forwarding.md

```diff
@@ -1,27 +1,37 @@
 # Conditional Forwarding
 
-After processing the data, conditional forwarding is doable based on the `Key` returned in the result.
+After processing the data, conditional forwarding is doable based on the `Tags` returned in the result. 
+Below is list of different logic operations that can be done on tags.
+- **and** - forwards the message if all the tags specified are present in Message's tags.
+- **or** - forwards the message if one of the tags specified is present in Message's tags.
+- **not** - forwards the message if all the tags specified is not present in Message's tags.
 
-For example, there's a UDF used to process numbers, and forward the result to different vertices based on the number is even or odd. In this case, you can set the `key` to `even` or `odd` in each of the returned messages, and define the edges as below:
+For example, there's a UDF used to process numbers, and forward the result to different vertices based on the number is even or odd. In this case, you can set the `tag` to `even-tag` or `odd-tag` in each of the returned messages,
+and define the edges as below:
 
 ```yaml
 edges:
   - from: p1
     to: even-vertex
     conditions:
-      keyIn:
-        - even
+      tags:
+        operator: or
```

Comment:
Need some opinions - do we encourage ppl to put `operator` all the time? Do we need to put a comment `# Optional, defaults to "or".`?

Reply:
+1, I think it's better to add that comment